apps/builder/app/builder/features/sidebar-left/panels/index.ts

```diff
@@ -5,7 +5,7 @@ import * as pages from "./pages";
 
 export const panels = {
   components,
+  pages,
```

Comment:
object fields order is not guaranteed. Better use the other structure

Reply:
not officially but its actually guaranteed, but lets make it more clear with a change